generators/client/templates/react/src/main/webapp/app/config/constants.ts.ejs

```diff
@@ -17,13 +17,12 @@
  limitations under the License.
 -%>
 const config = {
-  VERSION: process.env.VERSION,
-  SERVER_API_URL: process.env.SERVER_API_URL
+  VERSION: process.env.VERSION
 };
 
 export default config;
 
-export const SERVER_API_URL = '';
```

Comment:
This was declared twice (in `config` and below), moved to just one declaration.  Can put back in `config`, the import in other files would need changes.

Reply:
its ok